package.json

```diff
@@ -61,7 +61,7 @@
     "test:browser": "npm run build && playwright install --with-deps firefox chromium && vitest run --dir=tests/browser --browser --config=tests/browser/vite.config.mts"
   },
   "dependencies": {
-    "@types/node": "^22.10.2",
+    "@types/node": "*",
```

Comment:
what was the main reason to wildcard this dependency? Always looks scary to wildcard dependencies  like that, even if types

Reply:
(Or maybe it's easier to leave it for now, since the PR is already approved?)

Reply:
I'm going to merge it, and then we can potentially change it after.